src/test/java/uk/gov/pay/webhooks/util/DatabaseTestHelper.java

```diff
@@ -13,17 +19,163 @@ private DatabaseTestHelper(Jdbi jdbi) {
     public static DatabaseTestHelper aDatabaseTestHelper(Jdbi jdbi) {
         return new DatabaseTestHelper(jdbi);
     }
-    
-    public void addWebhookWithSubscription(String webhookExternalId, String serviceExternalId, String callbackUrl, String gatewayAccountId) {
+
+    public void addWebhook(String webhookExternalId, String serviceExternalId, String callbackUrl, String gatewayAccountId) {
         jdbi.withHandle(h -> h.execute("INSERT INTO webhooks VALUES (1, '2022-01-01', '%s', 'signing-key', '%s', false, '%s', 'description', 'ACTIVE', '%s')".formatted(webhookExternalId, serviceExternalId, callbackUrl, gatewayAccountId)));
-        jdbi.withHandle(h -> h.execute("INSERT INTO webhook_subscriptions VALUES (1, (SELECT id FROM event_types WHERE name = 'card_payment_succeeded'))"));
-        jdbi.withHandle(h -> h.execute("INSERT INTO webhook_subscriptions VALUES (1, (SELECT id FROM event_types WHERE name = 'card_payment_refunded'))"));
     }
 
-    public void truncateAllData() {
+    public void addWebhookMessageLastDeliveryStatusIsConsistent(String serviceExternalId, String gatewayAccountId, int wireMockPort) {
+        jdbi.withHandle(h -> h.execute("""
+                    INSERT INTO webhooks VALUES
+                    (1, '2022-01-01', 'webhook-external-id-succeeds', 'signing-key', '%s', false, 'http://localhost:%d/a-working-endpoint', 'description', 'ACTIVE', '%s'),
+                    (2, '2022-01-01', 'webhook-external-id-fails', 'signing-key', '%s', false, 'http://localhost:%d/a-failing-endpoint', 'description', 'ACTIVE', '%s')
+                """.formatted(
+                serviceExternalId, wireMockPort, gatewayAccountId,
+                serviceExternalId, wireMockPort, gatewayAccountId)
+        ));
+    }
```

Comment:
This is a bit specific. It looks like this method is inserting two webhooks with different webhook external IDs. So we could have…

```java
public void addWebhook(String serviceExternalId, String gatewayAccountId, String webhookExternalId, int wireMockPort) {
    jdbi.withHandle(h -> h.execute("""
            INSERT INTO webhooks VALUES
            (1, '2022-01-01', '%s', 'signing-key', '%s', false, 'http://localhost:%d/a-working-endpoint', 'description', 'ACTIVE', '%s'))
            """.formatted(webhookExternalId, serviceExternalId, wireMockPort, gatewayAccountId);
}
```

… and just call it twice from the test with different arguments.